examples/gazebo_scene_viewer/SceneManagerPrivate.hh

```diff
@@ -114,27 +114,27 @@ namespace ignition
 
       private: NewSceneManager *newSceneManager;
 
-      private: gazebo::transport::NodePtr transportNode;
+      private: sim::transport::NodePtr transportNode;
```

Comment:
these should stay `gazebo` as this demo uses gazebo-classic transport to get its scene info and render the scene using gz-rendering

Reply:
Ahh. I reverted these changes.

Reply:
I still see `sim::`, maybe the changes were not pushed?

Reply:
more `sim::` to `gazebo::` changes in [df38c4e](https://github.com/gazebosim/gz-rendering/pull/705/commits/df38c4e6cc8344c999f0d9f5ba401c9b4067f22b)